### ISSUE TITLE
Null element stack trace in applynumericalstyles

### ISSUE THREAD

**Issue by JolivetteSax** [sweetalert2/sweetalert2-react-content#110]
Upgrading from SA2 v.7 and this package v.1 to the latest in npm gives a stack error

"TypeError: Cannot read property 'style' of null"  
applyNumericalStyle  
node_modules/sweetalert2/dist/sweetalert2.all.js:564  

The stack trace points to this line of code:
       elem.style.removeProperty(property);
 
Latest from npm install gives:
    "sweetalert2": "^9.10.5",
    "sweetalert2-react-content": "^3.0.1",

There must be some regression because going back to the old version works without any problem. Even cutting and pasting the simplest code from the readme doesn't work. Other deps I'm mixing with this include:
   "react": "^16.8.6",
    "react-bootstrap": "^1.0.0-beta.17",
    "react-dom": "^16.8.6",

**Comment by zenflow** [comment]
@JolivetteSax Thanks for reporting. 

In the future, please include exact version numbers, because new releases can be made quite frequently. In fact, I think there have already been (a) new release(s) since you made these notes, because line 564 in https://unpkg.com/browse/sweetalert2@9.10.5/dist/sweetalert2.all.js doesn't have `elem.style.removeProperty(property);`. 

I found the line with `elem.style.removeProperty(property);` and examined all the call sites to the function that contains it, but I can't see how that error could possibly be caused by an interaction with **this** package.

Can you please create a **minimal** reproduction of the issue so that we can investigate? You can use [codepen](https://codepen.io/limonte/pen/JjoWRMx) or [codesandbox](https://codesandbox.io/) or anything like that. Just start with exactly what you have, and then  gradually remove all the code that doesn't remove the error. (That may include removing the use of this library so that you're only using sweetalert2.)

Thanks!

**Comment by JolivetteSax** [comment]
I'm sorry, but I don't understand your comment about version numbers. I cut and paste the package.json result from the command to npm install both packages, and both are the latest versions tagged in npm and those are the versions I got. 

This is a stack pop I'm getting from vanilla react and a vanilla install of sweetalert2 and sweetalert2-react-content. 

I fixed it by reverting to an old version I used in a previous project and everything works fine there, however if you can't replicate I can set up a code pen, but I'll have to get to it tomorrow when I fire up my dev rig again.

Thanks for your help!

**Comment by zenflow** [comment]
> I'm sorry, but I don't understand your comment about version numbers. I cut and paste the package.json result from the command to npm install both packages, and both are the latest versions tagged in npm and those are the versions I got.

My bad. I was looking at your first statement ("Upgrading from SA2 v.7 and this package v.1 to the latest in npm gives a stack error") and somehow overlooked the exact version numbers copy-pasted from your package.json. It's strange that line 564 in https://unpkg.com/browse/sweetalert2@9.10.5/dist/sweetalert2.all.js doesn't have `elem.style.removeProperty(property);`, but it doesn't matter, as long as we are using the versions you specified. I will assume that the text `node_modules/sweetalert2/dist/sweetalert2.all.js:564` was recorded when testing with a previous version or something.

**Comment by zenflow** [comment]
> I fixed it by reverting to an old version I used in a previous project and everything works fine there, however if you can't replicate I can set up a code pen, but I'll have to get to it tomorrow when I fire up my dev rig again.

@JolivetteSax No, I wasn't able to replicate the error given just `sweetalert2@9.10.5` & `sweetalert2-react-content@3.0.1` and "try any simple code from the README".  I tried with a few different versions of react & react-dom (latest, 16.8.6, and another more recent version). react-bootstrap really shouldn't factor into it, especially if it's not being used in the simple code. Feel free to build onto this https://glitch.com/edit/#!/jade-cultured-wasabi

**Comment by JolivetteSax** [comment]
This is.. weird? I'll try to figure this out manana. I mean, it must be something dumb that I was doing. I'm really sorry if so, but I'll try to run it down in case there's something subtle that I can identify for you.

**Comment by limonte** [comment]
I believe this is the issue in the core library, one of our users has it in Electron project:

![image](https://user-images.githubusercontent.com/6059356/77629756-b0a7a680-6f52-11ea-8d41-f1f66fab2898.png)
